packages/charting/src/components/Legends/Legends.styles.ts

```diff
@@ -48,11 +48,14 @@ export const getStyles = (props: ILegendStyleProps): ILegendsStyles => {
       },
       width: '12px',
       height: '12px',
-      backgroundColor: props.colorOnSelectedState,
+      backgroundColor: props.stripePattern ? '' : props.colorOnSelectedState,
       marginRight: '8px',
       border: '1px solid',
       borderColor: props.borderColor ? props.borderColor : theme?.semanticColors.buttonBorder,
-      opacity: props.colorOnSelectedState === palette.white ? '0.6' : '',
+      opacity: props.colorOnSelectedState === palette.white ? '0.6' : props.opacity ? props.opacity : '',
+      backgroundImage: props.stripePattern
+        ? `repeating-linear-gradient(135deg, transparent, transparent 3px, ${props.colorOnSelectedState} 1px, ${props.colorOnSelectedState} 4px)`
```

Comment:
did you check this css in window high contrast mode, for both, white and black theme?

Reply:
I'm not sure how to properly test this. Do you have any references I can refer to? Thank you!

Reply:
in window -> search -> type `ease of access high contrast settings` and switch on the high contrast . 

then open the `edge chromium` browser and open you application. check if the colors are showing properply.

if not then you need add some css in `[highcontrasSelector]` see this file only for reference , for example styles for the `triangle`

Reply:
I checked and the legends are behaving like the existing behavior. However, as you can see, the legends do not have any colors. I checked this behavior with another code base that uses fluent ui and those legends also do not have color when in high contrast mode. 
![image](https://user-images.githubusercontent.com/69655270/93274525-68ce4300-f76f-11ea-8d5f-ade86a51f2d6.png)